client/state/data-layer/http-data/getters.js

```diff
@@ -0,0 +1,38 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import makeJsonSchemaParser from 'lib/make-json-schema-parser';
+import { http as rawHttp } from 'state/http/actions';
+import { requestHttpData } from 'state/data-layer/http-data/common';
+
+export const requestGeoLocation = () =>
+	requestHttpData(
+		'geo',
+		rawHttp( {
+			method: 'GET',
+			url: 'https://public-api.wordpress.com/geo/',
+		} ),
+		{
+			fromApi: makeJsonSchemaParser(
```

Comment:
Creating a JSON schema parser is quite an expensive operation. And we create a new one from scratch on every `requestGeoLocation` call, don't we? Selectors are the kind of code that makes sense to aggresively optimize. Can we refactor the `requestHttpData( 'geo' )` call to be a simple `Map` getter?

Reply:
well I thought they weren't expensive ever since we made them lazy. I agree on the point about creating them on every call